### PR TITLE
typo fix on line 112 of timeboard,py  pull_all command 

### PR DESCRIPTION
**Pull request by jhotta**

### CODE REVIEW

File: datadog/dogshell/timeboard.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ def _title_to_filename(title):
             return no_punct.replace(" ", "_").replace("-", "_").strip("_")
 
         format = args.format
-        res = api.Timeboard.get_all()()
+        res = api.Timeboard.get_all()
```
Comment (yannmh) [inline datadog/dogshell/timeboard.py]: Thanks for fixing the typo :pencil2:  !

**Comment by yannmh** [comment]
Thanks for the fixes @jhotta ! 

Rather than having the `_pretty_json` method defined in multiple sections of the code, what do you think of moving it to a file like `datadog/util/format.py` directory?

**Comment by remh** [comment]
@yannmh yup this should be refactored. Can you do it so that we release this bug in a timely fashion please ?

**Comment by yannmh** [comment]
Closing in favor of #92. Thanks @jhotta

**Pull request closed without merging**